app/bundles/PageBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -560,13 +572,12 @@ public function onBuildJsForTrackingEvent(BuildJsEvent $event)
                 if (typeof ga  !== 'undefined' && typeof events.google_analytics !== 'undefined') {
                  var e = events.google_analytics; 
                      for(var i = 0; i < e.length; i++) {
-                         if(typeof e[i]['action']  !== 'undefined' && typeof e[i]['label']  !== 'undefined' )
-                             	ga('send', {
-                                    hitType: 'event',
-                                    eventCategory: e[i]['category'],
-                                    eventAction: e[i]['action'],
-                                    eventLabel: e[i]['label'],
-			                     });
+                         if(typeof e[i]['action']  !== 'undefined' && typeof e[i]['label']  !== 'undefined' ) {
+                            gtag('event', e[i]['action'], {
```

Comment:
This change is missing from https://github.com/mautic/mautic/pull/12421. Is that on purpose?

Reply:
Nope I just noticed that the facebook code uses analytics.js. Ill need to add this to the v5 pr too and test it properly.

Reply:
I have updated the analytics.js event to gtag for the v5 one too and tested it locally.